### DISCUSSION TITLE
Cannot run headless installation becasue shrinkwrap.yaml is not up to date

### DISCUSSION THREAD

**Discussion by lennaylennay2** [pnpm/pnpm#5104] [Q&A]
I'm using an old version of pnpm for an old project that im not able to entrierly update. 

I don't believe this to be a bug but has anybody run into this kind of problem before? 
 13>PNPMRestore:
         Performing headless installation
     3>LocalizeResources:
         Localization finished for 'id-ID(1057)'
         Start localization for 'uk-UA(1058)'
    13>PNPMRestore:
         **ERRORΓÇë Cannot run headless installation because shrinkwrap.yaml is not up-to-date with package.json**
         at <anonymous>  ΓÇªmodules/@pnpm/headless/lib/index.js:103
         at next
         at fulfilled    ΓÇªe_modules/@pnpm/headless/lib/index.js:4
    13>D:\MailApp\CRM.Client.MailAppShim\src\build\targets\NodeModules.targets(23,3): error MSB3073: The command "D:\MailAppCache\Crm.Node.js.10.16.0\node.exe D:\MailAppCache\PNPMpkg.PNPM.2.9.0\content\lib\bin\pnpm.js install --store D:\MailAppCache\.pnpm-store --frozen-shrinkwrap" exited with code 1. [D:\MailApp\C
       RM.Client.MailAppShim\src\OutlookModuleExtension\OutlookModuleExtension.csproj]

**Comment by zkochan** [comment]
Try `pnpm install --no-frozen-shrinkwrap`

or remove the lockfile and run pnpm install
